bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -156,24 +153,6 @@ public void run() {
     connectionStartupExecutor.execute(new Runnable() {
       @Override
       public void run() {
-        // The first invocation of BigtableServiceGrpc.CONFIG is expensive.
```

Comment:
Don't we need the first call to data and table service? Is this change part of the PR?